main/bash-completion/template.py

```diff
@@ -13,6 +13,10 @@
 license = "GPL-2.0-or-later"
 url = "https://github.com/scop/bash-completion"
 source = f"{url}/releases/download/{pkgver}/{pkgname}-{pkgver}.tar.xz"
-sha256 = "73a8894bad94dee83ab468fa09f628daffd567e8bef1a24277f1e9a0daf911ac"
+sha256 = "c5f99a39e40f0d154c03ff15438e87ece1f5ac666336a4459899e2ff4bedf3d1"
 # what's the point (needs a truckload of checkdepends too)
 options = ["!check"]
+
+
+def post_install(self):
+    self.rm(self.destdir / "etc/bash_completion.d", recursive=True)
```

Comment:
this technically places a compat function thing in there but idk if it's that important

Reply:
you can pass `compatdir=/usr/share/bash-completion/completions` in `make_install_args` like i do in https://github.com/chimera-linux/cports/pull/1826 i guess

Reply:
hmm actually then you'd need to either patch `bash_completion` to use that or set the `BASH_COMPLETION_COMPAT_DIR` envvar somewhere to use it, so i guess we can just remove the file if it's not important

Reply:
yeah will just leave it out for now i guess

Reply:
ah no, _get_cword is compat and needed by some things

Reply:
bleh